packages/paste-website/src/components/tokens-list/TokensListFilter.tsx

```diff
@@ -9,6 +9,8 @@ import {Grid, Column} from '@twilio-paste/grid';
 import {Select, Option} from '@twilio-paste/select';
 import {useUID} from '@twilio-paste/uid-library';
 
+import {SITE_MASTHEAD_HEIGHT} from '../../constants';
```

Comment:
suggestion: can we rename it from MASTHEAD to TOPBAR please?

Reply:
Yes, good idea